webservice/src/main/java/com/hortonworks/streamline/webservice/StreamlineApplication.java

```diff
@@ -272,6 +272,15 @@ private void registerResources(StreamlineConfiguration configuration, Environmen
             initConfig.put(Constants.CONFIG_AUTHORIZER, authorizer);
             initConfig.put(Constants.CONFIG_SECURITY_CATALOG_SERVICE, securityCatalogService);
             initConfig.put(Constants.CONFIG_SUBJECT, subject);
+            if (configuration.getHttpProxyUrl() == null || configuration.getHttpProxyUrl().isEmpty()) {
```

Comment:
to catch old configs it should be
```
configuration.getHttpProxyUrl() != null && !configuration.getHttpProxyUrl().isEmpty()
```

Reply:
To catch old configs it should should be initConfig.getProperties().get("proxyUrl"); I will change around the logic for detecting and exit in that case.

Reply:
I changed my mind and instead of exiting we will use top level configs.